src/utils/apiResponse.utils.ts

```diff
@@ -0,0 +1,29 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
+export type ControllerResponse = {
+  isSuccess: number;
+  statusCode: number;
+  message: string;
+  data?: any;
+  error?: any;
```

Comment:
PullProofAI

It's recommended to avoid using `any` type. Consider specifying a more specific type or interface for `data` and `error`.


<!-- This is an auto-generated comment by PullProofAI -->